python/ray/data/read_api.py

```diff
@@ -1772,20 +1772,47 @@ def from_spark(
 @PublicAPI
 def from_huggingface(
     dataset: Union["datasets.Dataset", "datasets.DatasetDict"],
-) -> Union[MaterializedDataset]:
+) -> Union[MaterializedDataset, Dict[str, MaterializedDataset]]:
     """Create a dataset from a Hugging Face Datasets Dataset.
 
     This function is not parallelized, and is intended to be used
     with Hugging Face Datasets that are loaded into memory (as opposed
     to memory-mapped).
 
+    Example:
+        >>> import ray
+        >>> import datasets
+        >>> hf_dataset = datasets.load_dataset("tweet_eval", "emotion")
+        >>> ray_ds = ray.data.from_huggingface(hf_dataset)
+        >>> ray_ds
+        {'train': MaterializedDataset(
+            num_blocks=1,
+            num_rows=3257,
+            schema={text: string, label: int64}
+        ), 'test': MaterializedDataset(
+            num_blocks=1,
+            num_rows=1421,
+            schema={text: string, label: int64}
+        ), 'validation': MaterializedDataset(
+            num_blocks=1,
+            num_rows=374,
+            schema={text: string, label: int64}
+        )}
+        >>> ray_ds = ray.data.from_huggingface(hf_dataset["train"])
+        >>> ray_ds
+        MaterializedDataset(
+            num_blocks=1,
```

Comment:
Is `num_blocks` gaurenteed to be 1? If not, you may want to use ellipsis to make the test determinsitic. https://docs.ray.io/en/master/ray-contribute/writing-code-snippets.html#ignoring-doctest-style-outputs

Reply:
yeah it is guaranteed to be 1 until we have distributed read support for `from_huggingface`